.vscode/tasks.json

```diff
@@ -1,120 +1,137 @@
-            "options": {
-                "cwd": "src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "CompileLibrary",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "WatchLibraryAndCompileExtension",
-            "type": "npm",
-            "script": "watch",
-            "problemMatcher": "$tsc-watch",
-            "options": {
-                "cwd": "src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/"
-            },
-            "isBackground": true,
-            "presentation": {
-                "reveal": "silent"
-            },
-            "dependsOn": [
-                "CompileExtension"
-            ]
-        },
-        {
-            "label": "CompileExtension",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            },
-            "dependsOn": [
-              "CompileLibrary"
-            ]
-        },
-        {
-            "label": "CompileUnitTests",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Test/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "CompileGrammarTests",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.Grammar.Test/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            }
-        },
-        {
-            "label": "CompileFunctionalTest",
-            "command": "dotnet",
-            "args": [
-                "build"
-            ],
-            "options": {
-                "cwd": "src/Razor/test/VSCode.FunctionalTest/"
-            },
-            "problemMatcher": "$tsc-watch",
-            "presentation": {
-                "reveal": "silent"
-            },
-            "dependsOn": [
-                "CompileExtension"
-            ]
-        },
-    ]
+  "version": "2.0.0",
```

Comment:
Not sure why the diff didn't print nicely here but the main change is prefacing all the `cwd` values with `${workspaceFolder}` to ensure that paths get resolved correctly when opening aspnetcore-tooling in a dev container.

Reply:
GitHub tip for reviewers: add `?w=1` at the end of the url to only show diff if the line has changes other than whitespace.